test/build.sh

```diff
@@ -16,8 +16,9 @@ cmakeConfigure()
     local CMAKEARGS=(..
            "-DCMAKE_Fortran_COMPILER=${fcomp}"
            "-DCMAKE_INSTALL_PREFIX=${idir}"
-           "-DCMAKE_CXX_COMPILER=g++"
-           "-DCMAKE_C_COMPILER=gcc"
+           "-DCMAKE_CXX_COMPILER=CC"
+           "-DCMAKE_C_COMPILER=cc"
```

Comment:
I don't know, actually. I have only experience with the other Cray machines, not with Kesch. How do you compile with MPI in Kesch?